aggregation_model: 1
alpha: 0
area_exploration: 10000
beta: 10
number_of_auvs: 2
number_of_robots: 2
pickle_path: /catkin_ws/src/multi_robot_system/missions/pickle/3AUVs/10000.pickle
w1: 0.4
w2: 0.4
w3: 0.2
w4: 0
